test/staging/FundMe.staging.test.ts

```diff
@@ -1,34 +1,34 @@
-import { assert } from "chai"
-import { ethers, network } from "hardhat"
-import { developmentChains } from "../../helper-hardhat-config"
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
-import { FundMe } from "../../typechain-types"
+import { ethers, network } from 'hardhat'
+import { FundMe } from '../../typechain-types'
+import { SignerWithAddress } from '@nomicfoundation/hardhat-ethers/signers'
+import { assert } from 'chai'
+import { developmentChains } from '../../helper-hardhat-config'
 
 developmentChains.includes(network.name)
     ? describe.skip
-    : describe("FundMe Staging Tests", async function () {
+    : describe('FundMe', async function () {
           let fundMe: FundMe
           let deployer: SignerWithAddress
 
-          const sendValue = ethers.utils.parseEther("0.1")
-          beforeEach(async function () {
+          const sendValue = ethers.parseEther('1')
+
+          beforeEach(async () => {
```

Comment:
Using `async ()` instead of `async function ()` is actually a bit of an anti-pattern. The keyword `this` is wonky with `async ()` instead of `async function ()`